src/ResourceManager/Dns/Commands.Dns/Commands.Dns.csproj

```diff
@@ -115,6 +121,10 @@
       <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
       <Name>Commands.ResourceManager.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Network\Commands.Network\Commands.Network.csproj">
```

Comment:
Use this project instead: https://github.com/Azure/azure-powershell/blob/2001d5053dfda2cbc3d9a04ba826b042311d6964/src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj#L343